src/network_analyzer.cpp

```diff
@@ -1178,6 +1179,7 @@ void NetworkAnalyzer::goertzel()
 		}
 	}
 
+	Q_EMIT sweepStart();
```

Comment:
Is this sweepStart signal used anywhere?

Reply:
It is now :stuck_out_tongue_winking_eye: